File: DAY 16/MemoryPass.py
list_a = [10, 20, 30]
list_b = list_a
print(f"Initial list_a: {list_a}")
list_b[0] = 99
print(f"list_a after list_b mutation: {list_a}")
print(f"Are they the same object? {list_a is list_b}")

File: DAY 16/MemoryPass.java
class NumberWrapper {
    int value;
    NumberWrapper(int val) {
        this.value = val;
    }
}
public class MemoryPass {
    public static void main(String[] args) {
        NumberWrapper nodeA = new NumberWrapper(10);
        NumberWrapper nodeB = nodeA; 
        System.out.println("Initial NodeA Value: " + nodeA.value);
        nodeB.value = 50;
        System.out.println("NodeA Value after NodeB mutation: " + nodeA.value);
    }
}